map.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./src/home/index.css">
+    <link rel="stylesheet" href="./src/map/map.css">
+    <title>Life Navigation</title>
+</head>
+<body>
+    <header>
+        <title>You wake up,</title>
+        <img src="./assets/spoon.png" alt="">
+        <!--we place the text outside of the span so it doesn't get overwritten-->
+        name: <span id="name"></span>
+        disability: <span id="disability"></span>
+        spoons: <span id="spoons"></span>
+        time:<span id="time"></span>
+    </header>
+
+    <main>
+        <h1>Pick a goal for today:</h1>
+
+        <div id="quests">
+            <!--TODO: DELETE THIS LINE <a class="quest" href="./quest.html">Quest 1</a> -->
+        </div>
+    </main>
+
+    <section>
+        <span class="completed">Completed Quest</span>
+    </section>
+
+    <script type="module" src="./src/map/map.js"></script>
+</body>
+</html>
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<!-- <main>
```

Comment:
Don't leave commented out code in projects. If you want to save it, I'd suggest moving it another project, that you can use as a storage place.